temporalio/worker/workflow_sandbox/_restrictions.py

```diff
@@ -933,7 +933,8 @@ def __init__(
 
         def bind_f(instance: _RestrictedProxy, obj: Any) -> Callable:
             def i_op(self: Any, other: Any) -> _RestrictedProxy:
-                f(self, other)  # type: ignore
+                # TODO: wat
+                f(self, other)  # type: ignore # noqa: F821
```

Comment:
I do not know what I was thinking here, but it's likely a bug. Would definitely welcome a test that covers this and fixes it to do what it is supposed to.

Reply:
👍 https://github.com/temporalio/sdk-python/issues/772